tools/rcverify.sh

```diff
@@ -212,7 +214,15 @@ packageJsonCheckVersion "$DIR/$BASE/package.json" $V
 printf "scanning package-lock.json for version match..."
 packageJsonCheckVersion "$DIR/$BASE/package-lock.json" $V
 
-echo $(tput setaf 6)
-echo run the following command to remove the scratch space:
-echo "  rm -rf '$DIR'"
-echo $(tput sgr0)
+if [ "$REMOVE_DIR" = true ]; then
```

Comment:
```suggestion
if [ "$REMOVE_DIR" = "cleanup" ]; then
```

only because as a positional argument `true` is ambiguous.
feel free to ignore the suggestion and merge.